### ISSUE TITLE
Text/icon flickering in URL bar

### ISSUE THREAD

**Issue by Villi00** [Naezr/ShyFox#211]
when hovering over url bar on hidden url bar style the contents in url bar starts to flicker when moving moving anywhere and doesnt stop until we try to write in the box. im using latest commit.
attaching a video of the bug 

things i tried :
reinstalled FF(normal)
reinstalled SHYFOX

https://github.com/user-attachments/assets/b2d79415-c32a-4dab-9aa2-68504eeff9e3

**Comment by Halbour** [comment]
I have the same exact problem. 
not only that, opening a new tab and then trying to type to search stuff doesn't work anymore, too.

**Comment by Villi00** [comment]
> I have the same exact problem. not only that, opening a new tab and then trying to type to search stuff doesn't work anymore, too.

yeah same problemi thought i was having hallucinations that the new tab opened that way in the past xD . hopefully he will fix it :)

**Comment by MorseShoes** [comment]
Yup! I have the same issue as well. Sometimes the bar even pops out like this
![URL Bar popping out](https://github.com/user-attachments/assets/c92dbc31-c0f8-4dc3-bbce-2b26ce1d8f39)

**Comment by xxxtrashcion** [comment]
I posted a possible fix here
https://github.com/Naezr/ShyFox/issues/190#issuecomment-4059305086

**Comment by TechRedByte** [comment]
> yeah same problemi thought i was having hallucinations that the new tab opened that way in the past xD . hopefully he will fix it :)

Sadly the developer abandoned this project long ago. 😞
